Frameworks/UIKit/UISegment.mm

```diff
@@ -192,6 +185,38 @@ - (id)drawRect:(CGRect)inRect {
 
     bounds = [self bounds];
 
+    id bgColor, textColor;
+
+    if (isOSTarget(@"7.0") && _selected == 0 && _tintColor != nil) {
+        textColor = _tintColor;
+        bgColor = [UIColor whiteColor];
+    } else {
```

Comment:
I left it there since I was unsure why it was there to start with.

I had built and run woccatalog targetting ios7 and ios9 and it seemed to look visually identical but I didn't feel comfortable removing it just in case there was a non-obvious reason that ios7 was singled out.